src/docfx/SubCommands/MetadataMerger.cs

```diff
@@ -104,6 +109,53 @@ private static void MergeToc(MetadataMergeParameters parameters, string outputBa
                 YamlMime.TableOfContent);
         }
 
+        private void InitializeTagsRecord(FileModel model)
+        {
+            var content = model.Content as PageViewModel;
+            if (content != null)
+            {
+                var metadata = content.Metadata;
+                object tagsObj;
+                if (metadata != null && metadata.TryGetValue("tags", out tagsObj))
+                {
+                    var tags = tagsObj as object[];
+                    if (tags != null)
+                    {
+                        var tagsList = tags.Select(tag => tag as string).Where(tag => tag != null).ToArray();
```

Comment:
i see

---
In reply to: [89735853](https://github.com/dotnet/docfx/pull/931#discussion_r89735853) [](ancestors = 89735853)